js/app.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() { 
+
+
+	//Hide 'Read More' text and 'Learn More' text initially
+ 	 $('#show-this-on-click', '#learnmoretext').hide ();
+
+ 	// setup the click handlers for 'Read more' and 'Learn More'
+ 	 $('#readMore').click('#show-this-on-click');
+  	 $('#learnMore').click('#lxearnmoretext');
```

Comment:
Lines 5, 8, and 9 have problems.

```
$('#show-this-on-click', '#learnmoretext').hide ();
```
You can't have a space between the words `hide` and the `()`. `hide` is a function and in order to call it you must write it like such `.hide()`.

The two `.click` events also won't work. the thing you pass to the `click()` (ie the thing you put inside the parenthesis must be a function aka event handler. Putting a string in there won't do anything).